src/main/MQ2ImGuiConsole.cpp

```diff
@@ -27,1504 +27,1638 @@
+		mq::imgui::HelpMarker("This feature allows you to hide MacroQuest Menu Bar.");
 
-	ImGui::NewLine();
+		// Setup Sliders 
+		ImGui::PushStyleVar(ImGuiStyleVar_FramePadding, ImVec2(2.0f, 1.0f));
+		ImGui::PushItemWidth(120);
 
-	if (ImGui::Button("Clear Saved Console Settings"))
-	{
-		s_consoleVisibleOnStartup = false;
-		WritePrivateProfileBool("MacroQuest", "ShowMacroQuestConsole", s_consoleVisibleOnStartup, mq::internal_paths::MQini);
+		ImGui::Text("Alpha Normal");
+		if (ImGui::SliderInt("##Alpha Normal", &s_alphaConsoleNormal, 0, 100))
+		{
+			WritePrivateProfileInt("MQConsole", "AlphaNormalOnStartup", s_alphaConsoleNormal, mq::internal_paths::MQini);
+		}
+
+		ImGui::Text("Alpha Fade");
+		if (ImGui::SliderInt("##Alpha Fade", &s_alphaConsoleFade, 0, 100))
+		{
+			WritePrivateProfileInt("MQConsole", "AlphaFadeOnStartup", s_alphaConsoleFade, mq::internal_paths::MQini);
+		}
+
+		ImGui::Text("Console Font Size");
+		if (ImGui::SliderInt("##ConsoleFontSize", &s_consoleFontSize, 8, 40))
+		{
+			WritePrivateProfileInt("MQConsole", "ConsoleFontSize", s_consoleFontSize, mq::internal_paths::MQini);
+		}
+
+		ImGui::PopItemWidth();
+		ImGui::PopStyleVar();
+
+		// Select UI font for console
+		ImGui::Text("UI Fonts Selector");
+		ImGui::ShowFontSelector("##FontsSelector");
+
+		// Check If UI Font changes and save it
+		std::string fontstr = ImGui::GetFont()->GetDebugName();
+		std::string consoleDebugName = s_consoleFontName;
+		if (fontstr != consoleDebugName)
+		{
+			s_consoleFontName = ImGui::GetFont()->GetDebugName();
+			WritePrivateProfileString("MQConsole", "ConsoleFontOnStartup", s_consoleFontName, mq::internal_paths::MQini);
+			s_consoleFontSize = (int)ImGui::GetFontSize();
+			WritePrivateProfileInt("MQConsole", "ConsoleFontSize", s_consoleFontSize, mq::internal_paths::MQini);
+		}
+		ImGui::NewLine();
+
+		ImGui::NewLine();
+		if (ImGui::Button("Clear Saved Console Settings"))
+		{
+			s_consoleVisibleOnStartup = false;
+			WritePrivateProfileBool("MQConsole", "ShowMacroQuestConsole", s_consoleVisibleOnStartup, mq::internal_paths::MQini);
+			s_isConsoleLocked = false;
+			WritePrivateProfileBool("MQConsole", "LockConsoleWindow", s_isConsoleLocked, mq::internal_paths::MQini);
+			s_isTitleBarShown = false;
+			WritePrivateProfileBool("MQConsole", "NoTitleBar", s_isTitleBarShown, mq::internal_paths::MQini);
+			s_alphaConsoleNormal = 100;
+			WritePrivateProfileInt("MQConsole", "AlphaNormalOnStartup", s_alphaConsoleNormal, mq::internal_paths::MQini);
+			s_alphaConsoleFade = 100;
+			WritePrivateProfileInt("MQConsole", "AlphaFadeOnStartup", s_alphaConsoleFade, mq::internal_paths::MQini);
+			s_consoleFontSize = 13;
+			WritePrivateProfileInt("MQConsole", "ConsoleFontSize", s_consoleFontSize, mq::internal_paths::MQini);
+			s_consoleFontName = "lucon.ttf, 13px";
+			s_consoleFontSize = 13;
+			int fontNumber = FontNumberFromString(s_consoleFontName);
+			ImGui::GetIO().FontDefault = ImGui::GetIO().Fonts->Fonts[fontNumber];
```

Comment:
the console font is not the default font.

Reply:
you should detect if the font is not found and fall back to the default console font (mq::imgui::ConsoleFont)